utils/auth.js

```diff
@@ -25,4 +31,5 @@ export const redirectIfNotAuth = async ({ req, res, query }) => {
     }`,
   });
   res.end();
+  return {};
```

Comment:
Do we need this if it wasn't there before? Fine if we do, just working out how it works :)

Reply:
It just stops a warning from appearing which makes logs a tad easier to go through